### ISSUE TITLE
Security vulnerability based on jsonpath dependency

### ISSUE THREAD

**Issue by ehmicky** [json-schema-faker/json-schema-faker#492]
The `jsonpath` dependency uses `static-eval`, a dependency which is [flagged as vulnerable](https://www.npmjs.com/advisories/758) by npm.

Consequently `npm audit` reports a security vulnerability when installing `json-schema-faker` and suggests downgrading to `0.5.0-rc9`:

```
                                                                                
                       === npm audit security report ===                        
                                                                                
# Run  npm install json-schema-faker@0.5.0-rc9  to resolve 1 vulnerability
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Sandbox Breakout / Arbitrary Code Execution                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ static-eval                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ json-schema-faker                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ json-schema-faker > jsonpath > static-eval                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/758                             │
└───────────────┴──────────────────────────────────────────────────────────────┘


found 1 moderate severity vulnerability in 26166 scanned packages
  run `npm audit fix` to fix 1 of them.
```

Note that [a PR](https://github.com/dchester/jsonpath/pull/121) has been submitted to `jsonpath`. Unfortunately this repository seems dormant so I'm not sure whether this will be merged anytime soon.

**Comment by pateketrueke** [comment]
Would exists smaller/faster alternatives to the `jsonpath` module?

**Comment by ehmicky** [comment]
Last time I've checked unfortunately not anything that is maintained :/

Might be worth doing a new check though.

**Comment by dinvlad** [comment]
This should be fixed in `jsonpath` 1.0.1, according to SourceClear. Could you please bump the dep to 1.0.1?

**Comment by ehmicky** [comment]
I am closing because even without bumping the version, since `json-schema-faker` uses `^1.0.0`, new installs will pick `1.0.1`. I can confirm `npm audit` does not complain anymore on my machine.

Thanks for the bump and sorry for suggesting the project was dormant.
